script.js

```diff
@@ -0,0 +1,18 @@
+const menuButton = document.querySelector('#showMobileMenuBtn');
+const mobileMenu = document.querySelector('#mobileMenu');
+const closeButton = document.querySelector('#closeButton');
+const navLinksMobile = document.querySelectorAll('.nav-link-mobile');
+
+menuButton.addEventListener('click', () => {
+  mobileMenu.style.display = 'flex';
+});
+
+closeButton.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+});
+
+navLinksMobile.forEach((navLink) => {
+  navLink.addEventListener('click', () => {
+    mobileMenu.style.display = 'none';
+  });
+});
```

Comment:
- [ ] Great algorithm and concept team. Great work!! 👏 😎

Reply:
@Gambit142 thanks!!!